templates/prescription/general_list.html

```diff
@@ -84,7 +84,7 @@
 }
 
 function deleteDrug(d) {
-        let msg = 'Do you really want to delete ';
+        let msg = xl('Do you really want to delete ');
```

Comment:
Back to here but replace
let msg = xl('Do you really want to delete ');
with
let msg = {/literal}{xlj t='Do you really want to delete?'}{literal};

Reply:
@sjpadgett the code has been reverted.